code/script.js

```diff
@@ -1,47 +1,178 @@
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/drbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Eventlistner 
+form.addEventListener('submit', (event) => {
+  event.preventDefault()
+  playSound()
+  const input = document.getElementById('name-input');
+  if (input) {
+    showMessage(input.value, 'user')
+    inputText = input.value
+    document.getElementById('name-input').value = ''
+    setTimeout(() => { healthQuestion() }, 1500)
+  }
+
+  setTimeout(() => {
+    document.getElementById('loading-img').classList.remove('loading-img-off')
+    document.getElementById('loading-img').classList.add('loading-img') 
+  }, 300);
+
+  setTimeout(() => {
+    document.getElementById('loading-img').classList.remove('loading-img')
+    document.getElementById('loading-img').classList.add('loading-img-off')
+  }, 1000);
+})
+
+// Chat starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage('Hi 👋🏼 I am your Corona Dr. I will help you with you corona test / vaccine booking. What is your name?', 'bot')
 }
 
-// Set up your eventlisteners here
+// Question #2
+const healthQuestion = () => {
+  showMessage(`Great meeting you ${inputText}! Do you feel sick?`, 'bot')
+  form.innerHTML = `
+    <button id="continue" onclick='buttonValue("Yes")'>Yes</button>
+    <button id="quit" onclick='buttonValue("No")'>No</button> 
+  `
+  document
+    .getElementById('continue')
+    .addEventListener('click', () => {
+      showMessage('Yes', 'user');
+      setTimeout(() => { optionQuestion() }, 1500)
+    })
+
+  document
+    .getElementById('quit')
+    .addEventListener('click', () => {
+      showMessage('No', 'user');
+      setTimeout(() => { goodByeMsg() }, 1500)
+    })
+}
+
+// Question #3
+const optionQuestion = () => {
+  showMessage('Do you want to take a corona test or book a vaccine appointment?', 'bot')
+  form.innerHTML = `
+      <button id="corona" onclick='buttonValue("Corona test")'>Corona 🦠</button>
+      <button id="vaccine" onclick='buttonValue("Vaccine booking")'>Vaccine 💉</button>
+    `
+  document
+    .getElementById('corona')
+    .addEventListener('click', () => {
+      showMessage('Corona test', 'user');
+      setTimeout(() => { questionCity('city') }, 1500)
+    })
+
+  document
+    .getElementById('vaccine')
+    .addEventListener('click', () => {
+      showMessage('Vaccine booking', 'user');
+      setTimeout(() => { callenderBooking('calender') }, 1500)
+    })
+}
```

Comment:
your questions functions have very similar construction. To avoid repeating yourself you can create a template function and pass data like botMessages, or answer options as parameters. `showMessage` function were built this way. 🤓